### PR TITLE
refactor!: Use `extism:host/env` namespace for extism functions and `extism:host/user` for user-defined host functions

### PR DESCRIPTION
**Pull request by zshipko**
See #504 

Removes CI for the in-repo SDKs, we can remove the actual code closer to the 1.0 release.

### CODE REVIEW

**Comment by zshipko** [comment]
Since the SDKs have mostly been moved to new repos, I'm not going to update the failing SDK tests. Instead, maybe this PR should include the changes to remove the old SDKs from this repo, since we will likely merge this closer to the planned 1.0 release date.

File: runtime/src/plugin.rs [hunk 1]
```diff
@@ -2,6 +2,9 @@ use std::collections::BTreeMap;
 
 use crate::*;
 
+pub const EXTISM_ENV_MODULE: &str = "extism:env";
+pub const EXTISM_USER_MODULE: &str = "extism:user";
```
Comment (chrisdickinson) [inline runtime/src/plugin.rs:6]: *winces* Ack, sorry sorry. I think we might want to add an interface projection here, too: something like `extism:core/env` / `extism:core/user`. Mea culpa here – I ran into this with the [observe-api](https://github.com/dylibso/observe-sdk/pull/128/files#diff-0a8d99077aa2dacc5aa5e7d571411c1f496737d0707f978771f9d65cd0674469R3) component model support. (The `/foo` bit maps to an interface in WIT.)

Comment (zshipko) [reply runtime/src/plugin.rs:6]: Thanks for catching this!!! I think I will go with `extism:host/env` and `extism:host/user`

**Review by chrisdickinson — DISMISSED** [review]
LGTM!

**Pull request merged**